src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/IExportTypes.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Mvc.ApplicationParts
+{
+    public interface IExportTypes
```

Comment:
Yeah this current name says nothing about what it's for. Try to convey the semantics.